.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+Thanks for your contribution and we appreciate it a lot.
+
+## Motivation
+Please describe the motivation of this PR and the goal you want to achieve through this PR.
+
+## Modification
+Please briefly describe what modification is made in this PR.
+
+## BC-breaking
+Does the modification introduces changes that break the back-compatibility of the downstream repos?
+If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.
+
+## Use cases
+If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.
+
+## Checklist
+
+1. Pre-commit or other linting tools are used to fix the potential lint issues.
+2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
+3. If the modification has potential influence on downstream projects, this PR is tested with downstream projects, like MMDet or MMCls.
```

Comment:
this PR is tested with downstream projects -> this PR should be tested with downstream projects